keyboards/neopad/rev1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,170 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /*
+            LAYER 0 - MUSIC
+    /-----------------------------`
+    |  TO(1)  |  stop   |   mute  |	L ENC: prev/next song
+    |---------|---------|---------|
+    |   prev  |  play   |   next  |	R ENC: vol up/down
+    \-----------------------------'
+    */
+    [0] = LAYOUT(
+                TO(1),     KC_MSTP,  KC_MUTE,
+    		    KC_MPRV,   KC_MPLY,  KC_MNXT
+    ),
+
+    /*
+            LAYER 1 - MOVEMENT IN WINDOWS
+    /---------------------------------------`
+    |    TO(2)    | maximize | show desktop | L ENC: change desktop
+    |-------------|----------|--------------|
+    | move window | minimize | move window  | R ENC: change browser tab + change explorer window
+    \---------------------------------------'
+    */
+    [1] = LAYOUT(
+				TO(2),     		LGUI(KC_UP),    LGUI(KC_D),
+				LGUI(KC_LEFT),	LGUI(KC_DOWN),  LGUI(KC_RIGHT)
+    ),
+
+    /*
+            LAYER 2 - SHORTCUT
+    /------------------------------`
+    |  TO(3)  |   esc   | task man | L ENC: redo/undo
+    |---------|---------|----------|
+    |   cut   |   copy  |   paste  | R ENC: mouse wheel up/down
+    \------------------------------'
+    */
+    [2] = LAYOUT(
+                TO(3),     		KC_ESC,    		LCTL(LSFT(KC_ESC)),
+    		    LCTL(KC_X),		LCTL(KC_C),     LCTL(KC_V)
+    ),
+
+    /*
+            LAYER 3 - AUDACITY
+    /-----------------------------`
+    |  TO(0)  |   REC   |   canc  |	L ENC: pan right/left
+    |---------|---------|---------|
+    |   ctrl  |   play  |  pause  |	R ENC: mouse wheel up/down
+    \-----------------------------'
+    */
+    [3] = LAYOUT(
+                TO(0),     	LSFT(KC_R),    	KC_DEL,
+    		    KC_LCTL,	KC_SPC,         KC_P
+    ),
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    // If console is enabled, it will print the matrix position and status of each key pressed
+/*
+#ifdef CONSOLE_ENABLE
+    uprintf("KL: kc: %u, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
+#endif
+*/
+    return true;
+}
+
+void keyboard_post_init_user(void) {
+    // Customise these values to desired behaviour
+    // debug_enable = true;
+    // debug_matrix = true;
+    // debug_keyboard = true;
+    // debug_mouse = true;
+}
+
```

Comment:
```suggestion
```

Reply:
And this one seems good to me too... can you explain me please where are the differences?

Reply:
This code is all commented out, so it should be removed.

Reply:
Alright.
Removed